src/lava/magma/core/model/py/model.py

```diff
@@ -282,8 +282,9 @@ class PyLoihiProcessModel(AbstractPyProcessModel):
 
     """
 
-    def __init__(self, proc_params: ty.Optional["ProcessParameters"] = None):
-        super().__init__(proc_params=proc_params)
+    def __init__(self, proc_params: ty.Optional["ProcessParameters"] = None,
+                 loglevel: ty.Optional[int] = logging.WARNING):
```

Comment:
Is this complete? Don't we need to flow the loglevel when creating instances of processmodels during compilation?

Reply:
Do we? I'm really not sure. I got frustrated when I realized how entangled the loglevel stuff is throughout a million different places and gave up. I guess we could just close this as it's not really very important.